corehq/apps/case_search/models.py

```diff
@@ -0,0 +1,118 @@
+from django.db import models
+from jsonfield.fields import JSONField
+
+FUZZY_PROPERTIES = "fuzzy_properties"
+
+
+class CaseSearchConfig(models.Model):
+    """
+    Contains config for case search
+    """
+    class Meta:
+        app_label = 'case_search'
+
+    domain = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+        db_index=True,
+        primary_key=True
+    )
+    enabled = models.BooleanField(blank=False, null=False, default=False)
+    config = JSONField(default={})
```

Comment:
I see we do this everywhere in the code but I think it's bad practice. The default should be `dict` so that everything doesn't point to the same instance. https://docs.djangoproject.com/en/1.9/ref/models/fields/#default